LCAPI.Core/LCAPI.Core.csproj

```diff
@@ -10,13 +10,37 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all"/>
+        <PackageReference Include="BepInEx.Analyzers" Version="1.*" PrivateAssets="all" />
         <PackageReference Include="BepInEx.Core" Version="5.*" />
-        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*"/>
+        <PackageReference Include="BepInEx.PluginInfoProps" Version="1.*" />
         <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework.TrimEnd(`0123456789`))' == 'net'">
-        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all"/>
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" />
     </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Assembly-CSharp">
+      <HintPath>D:\SteamLibrary\steamapps\common\Lethal Company\Lethal Company_Data\Managed\Assembly-CSharp.dll</HintPath>
```

Comment:
We should use relative paths for this. Preferably a "Dependency" directory in which we copy the dll files.

Reply:
Hey just passing by checking out the LC modding community.

You can define a variable for the game's install directory in a .user file, then use that variable for the paths for all of your references here.

For example
```xml
<Reference Include="Assembly-CSharp">
    <HintPath>$(GameDir)\Lethal Company_Data\Managed\Assembly-CSharp.dll</HintPath>
</Reference>
```

That's how I typically do it in my BepInEx mods
For example
<https://github.com/megalon/nasb2-custom-music-mod/blob/master/NASB2CustomMusicMod.csproj>

The .user file has to be in the gitignore though, so the developer has to make the file themselves, but I just explain that in the readme
<https://github.com/megalon/nasb2-custom-music-mod/tree/master#-developing>

Reply:
This issue is resolved with my latest commit